src/bin/tattoo.ts

```diff
@@ -24,18 +24,24 @@ async function main() {
     mergeConfigFileOptions(cliOptions, loadConfigFileOptions(cliOptions));
     ensureGitHubToken(cliOptions);
     const runnerOptions: RunnerOptions = {
+      color: cliOptions['color'],
```

Comment:
once we move into the programatic runner interface, lets drop the 'on'/'off' in favor of true/false for the `color` option. Then you don't need to translate to a boolean inside the runner.